mediathread/main/views.py

```diff
@@ -811,7 +814,8 @@ def send_faculty_email(form, faculty_user):
             subject,
             body,
             settings.SERVER_EMAIL,
-            [faculty_user.email])
+            [faculty_user.email],
+            fail_silently=not settings.DEBUG)
```

Comment:
The way I'm reading this: fail_silently is set to True in production. Is that the intention?